src/index.js

```diff
@@ -6,9 +6,62 @@ import './images/turing-logo.png'
 import './images/login-background.jpg'
 import './images/silva-logo.png'
 import './images/user-page.png'
+import Customer from './Customer'
+import Manager from './Manager'
 
-$("#login-submit-button").on('click', loginHandler);
-$("#user-header-logout").on('click', logout);
+let usersData;
+let roomsData;
+let bookingsData;
+let customer;
+let manager;
+let id;
+
+$("body").on('click', "#user-header-logout", addloginHTML);
+$("body").on('click', "#login-submit-button", loginHandler);
+
+usersData = getData('users/users', 'users');
+roomsData = getData('rooms/rooms', 'rooms');
+bookingsData = getData('bookings/bookings', 'bookings');
+
+Promise.all([usersData, roomsData, bookingsData]).then((promise) => {
+  usersData = promise[0];
+  roomsData = promise[1];
+  bookingsData = promise[2];
+}).then(data => {
+  usersData = usersData.users;
+});
+
+function formatDate(date) {
+  var monthNames = [
+    "1", "2", "3",
+    "4", "5", "6", "7",
+    "8", "9", "10",
+    "11", "12"
+  ];
+  var day = date.getDate();
+  var monthIndex = date.getMonth();
+  var year = date.getFullYear();
+  return year + '/' + monthNames[monthIndex] + '/' + day;
+}
+
+  const date = formatDate(new Date());
+	const dateObject = new Date(date);
+	const options = {
+		weekday: 'long',
+		year: 'numeric',
+		month: 'long',
+		day: 'numeric'
+	}
+
+	const formattedDate = dateObject.toLocaleString('en', options);
```

Comment:
This formate date function is a nice touch, definitely makes for a better user experience to see the data written out like that. Good call!